src/components/petcard/generateview.tsx

```diff
@@ -0,0 +1,45 @@
+'use client';
+
+import Link from 'next/link';
+
+import { button } from '@/styles/ogoo';
+import { flexCol, flexColCenter } from '@/styles/ogoo/alignment.css';
+import { whiteText } from '@/styles/ogoo/colors.css';
+import { titleLg, titleMd } from '@/styles/ogoo/typography.css';
+import { cn } from '@/utils';
+
+type Props = {
+  questionNumber: string;
+  firstTitle: string;
+  secondTitle: string;
+  children: React.ReactNode;
+  nextLink: string;
+};
```

Comment:
```suggestion
interface Props {
  questionNumber: number;
  title: string;
  children: React.ReactNode;
  nextLink: string;
};
```

혹시 interface 대신 type으로 사용하신 이유가 있을까유? 👀 ㅎㅎ